scripts/graph_accelerometer.py

```diff
@@ -17,35 +17,50 @@
 def parse_log(logname, opts):
     with open(logname) as f:
         for header in f:
-            if not header.startswith('#'):
+            if header.startswith('#'):
+                continue
+            if header.startswith('freq,psd_x,psd_y,psd_z,psd_xyz'):
+                # Processed power spectral density file
                 break
-        if not header.startswith('freq,psd_x,psd_y,psd_z,psd_xyz'):
             # Raw accelerometer data
             return np.loadtxt(logname, comments='#', delimiter=',')
-    # Power spectral density data or shaper calibration data
-    opts.error("File %s does not contain raw accelerometer data and therefore "
-               "is not supported by graph_accelerometer.py script. Please use "
-               "calibrate_shaper.py script to process it instead." % (logname,))
+    # Parse power spectral density data
+    data = np.loadtxt(logname, skiprows=1, comments='#', delimiter=',')
+    calibration_data = shaper_calibrate.CalibrationData(
+            freq_bins=data[:,0], psd_sum=data[:,4],
+            psd_x=data[:,1], psd_y=data[:,2], psd_z=data[:,3])
+    calibration_data.set_numpy(np)
+    return calibration_data
 
 ######################################################################
 # Raw accelerometer graphing
 ######################################################################
 
-def plot_accel(data, logname):
-    first_time = data[0, 0]
-    times = data[:,0] - first_time
+def plot_accel(datas, lognames):
     fig, axes = matplotlib.pyplot.subplots(nrows=3, sharex=True)
-    axes[0].set_title("\n".join(wrap("Accelerometer data (%s)" % (logname,),
-                                     MAX_TITLE_LENGTH)))
+    axes[0].set_title("\n".join(wrap(
+        "Accelerometer data (%s)" % (', '.join(lognames)), MAX_TITLE_LENGTH)))
     axis_names = ['x', 'y', 'z']
+    for data, logname in zip(datas, lognames):
+        if isinstance(data, shaper_calibrate.CalibrationData):
+            raise error("Cannot plot raw accelerometer data using the processed"
```

Comment:
This `error()` is not defined. Did you meant `RuntimeError` or another exception?